redfish-core/include/utils/hw_isolation.hpp

```diff
@@ -275,20 +275,26 @@ inline void
                     else if (strcmp(dbusError->name,
                                     "xyz.openbmc_project.Common.Error."
                                     "Unavailable") == 0)
+                    {
+                        messages::serviceDisabled(
+                            aResp->res, "/redfish/v1/Systems/system/"
```

Comment:
Do we need a disabled Error?

Reply:
FYI, I posted the below response for this [comment](https://github.com/ibm-openbmc/bmcweb/pull/309/commits/edb2243cf318414bf0f80a58a12475e95ade25ed#r840688817) but, I could not see it in the PR so adding it here too.

IMO, an Unavailable error in the D-Bus level is fine for the backend application for this kind of use case.

But, we need to differentiate only on the Redfish side based on the context so I think we can go with it.

Still, you feel we need an error like 1-to-1 mapping i.e Redfish to D-Bus, I think we need to talk more from the design point of view i.e supporting Redfish message registry in the D-Bus or similar design since we can add arguments for the message registry but, (AFAIK) we can not do that in the current D-Bus error design (YAML file).

Reply:
@gtmills I think we don't need same naming convention followed for mapping D-bus error to redfish errors right?
As Ramesh explained seems ServiceDisabled makes more sense as service disabled and there no retry time interval associated in this use-case.
ServiceDisabled error also underlying maps to boost::beast::http::status::service_unavailable status code.
I don't think we need any changes here

Reply:
I have removed the code to handle the `Unavailable` error, we don't have support to enable the hardware isolation service through the Redfish for the user. Refer [here](https://github.com/ibm-openbmc/bmcweb/pull/309/commits/6d57fd3a3a494b92e14811bb616bbd0562914a89) to get more details.